src/modules/interactionServer.js

```diff
@@ -21,6 +21,18 @@ function postCommentsOptions(cardID, username, comment) {
   };
 }
 
+// function postLikes(cardID) {
+//   return {
+//     method: 'POST',
+//     body: JSON.stringify({
+//       item_id: cardID,
+//     }),
+//     headers: {
+//       'Content-type': 'application/json',
+//     },
+//   };
+// }
+
```

Comment:
It would be nice if we remove all the commented code after finish the work. :100: